lib/Dialect/FIRRTL/Transforms/LowerAnnotations.cpp

```diff
@@ -267,7 +293,15 @@ static const llvm::StringMap<AnnoRecord> annotationRecords{{
     {"circt.test", {stdResolve, applyWithoutTarget<true>}},
     {"circt.testLocalOnly", {stdResolve, applyWithoutTarget<>}},
     {"circt.testNT", {noResolve, applyWithoutTarget<>}},
-    {"circt.missing", {tryResolve, applyWithoutTarget<>}}
+    {"circt.missing", {tryResolve, applyWithoutTarget<>}},
+    // Grand Central Views/Interfaces Annotations
+    {serializedViewAnnoClass, {noResolve, applyGCTView}},
+    {viewAnnoClass, {noResolve, applyGCTView}},
+    {companionAnnoClass, {stdResolve, applyWithoutTarget<>}},
+    {parentAnnoClass, {stdResolve, applyWithoutTarget<>}},
+    {augmentedGroundTypeClass, {stdResolve, applyWithoutTarget<true>}},
+    // Miscellaneous Annotations
+    {dontTouchAnnoClass, {stdResolve, applyDontTouch}}
```

Comment:
I love how this allows us to factor out the scattering logic and move it closer to the passes that actually make use of it.